bookbotproject/conversationtree/migrations/0004_auto_20171019_2008.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-10-19 17:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('conversationtree', '0003_conversationtree_rootnode'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='conversationnode',
+            name='name',
+        ),
+        migrations.AddField(
+            model_name='conversationnode',
+            name='intent',
+            field=models.CharField(blank=True, max_length=50),
```

Comment:
Also this max_length field needs to be changed, I guess this is an autogenerated file, please dont forget to make necessary changes if you're going to change CharField length of "intent" field

Reply:
Oh thank you, I was about to forget this but thanks to you I did that now. O:)